### PR TITLE
add readthedocs support

### LINKED ISSUE #203: Create .readthedocs.yml for packages having documentation
If a package has documentation it is probably rendered on ReadTheDocs.
That service requires some configuration which can be automatically created/updated by `meta/config`.

See https://github.com/zopefoundation/zope.i18n/pull/58/files for a working example.

### PR DESCRIPTION
**Pull request by dhavlik**
Fixes #203

### CODE REVIEW

File: config/config-package.py [hunk 1]
```diff
@@ -328,6 +328,12 @@ def gitignore(self):
             ignore=git_ignore,
         )
 
+    def readthedocs(self):
+        self.copy_with_meta(
+            'readthedocs.yaml.j2', self.path / '.readthedocs.yaml',
```
Comment (icemac) [inline config/config-package.py:333]: A `git add` call is missing for this new file.

**Review by icemac — CHANGES_REQUESTED** [review]

File: config/config-package.py (hunk 1 above)
Comment (dhavlik) [reply config/config-package.py:333]: added

**Review by icemac — APPROVED** [review]
LGTM.

**Comment by icemac** [comment]
> How to use this is not documented in the README

I added https://github.com/zopefoundation/meta/pull/212 to fix this.

**Pull request merged**